WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -9704,6 +9704,10 @@
 /* Link to a WordPress.org page for the plugin */
 "WordPress.org Plugin Page" = "WordPress.org Plugin Page";
 
+/* Caption displayed during the login flow. */
+"wordpress.prologue.splash.caption" = "Write, edit, and publish
+from anywhere.";
+
```

Comment:
I was a bit surprised by having only one change in the strings but this is legit.

To verify:

- I replayed the automation on a fresh branch and got the same result
- [Compared `20.7` with `trunk`](https://github.com/wordpress-mobile/WordPress-iOS/commit/ccacbda17e759d5cd32894f61fadeff3297c64e6) and searched for `NSLocalizedStrings` usages. There were only three results and this was the only addition.

<img width="904" alt="image" src="https://user-images.githubusercontent.com/1218433/190952383-be9b2e7e-7179-4468-acb0-28fa0defe58f.png">

Reply:
And an interesting one, since it's one finally using reverse-DNS notation 💪  but also multiline string literals, which I didn't know were supported (aka parsed correctly) by `genstrings` (which afaik is based on regex… so I wouldn't have been so sure that multiline syntax would have been supported) 👍